contrib/initramfs/scripts/zfs

```diff
@@ -405,6 +405,8 @@ decrypt_fs()
 		ENCRYPTIONROOT="$(get_fs_value "${fs}" encryptionroot)"
 		KEYLOCATION="$(get_fs_value "${ENCRYPTIONROOT}" keylocation)"
 
+		echo -n "${ENCRYPTIONROOT}" > /run/zfs_fs_name
```

Comment:
Still an `echo -n` here.

Reply:
Removed, works correctly after tests, thans for reminding